src/components/UpdatePaymentDetailsPopup/Steps/Error.js

```diff
@@ -38,12 +38,12 @@ const Error = () => {
         <TitleStyled>{t('Oops, something went wrong')}</TitleStyled>
         <TextStyled>
           {error?.includes('Refused') ? (
-            <Trans i18nKey="update-payment-error-method">
+            <Trans i18nKey="update-payment-error.method">
               We weren’t able to update your payment method. <br /> Please try
               again.
             </Trans>
           ) : (
-            <Trans i18nKey="update-payment-error-details">
+            <Trans i18nKey="update-payment-error.details">
               We weren’t able to update your payment details. <br /> Please try
               again using different payment method.
             </Trans>
```

Comment:
the keys here are still missleading for me. I would use `update-payment-details` (as a feature), then dot and type of error, so: 
`update-payment-details.refused`
`update-payment-details.error`